packages/renderer/src/lib/kube/KubernetesDashboardGuideCard.svelte

```diff
@@ -0,0 +1,16 @@
+<script lang="ts">
+import { Button } from '@podman-desktop/ui-svelte';
+
+let {
+  title,
+  link,
+}: {
+  title: string;
+  link: string;
+} = $props();
+</script>
+
+<div class="flex flex-col gap-4 p-4 bg-[var(--pd-invert-content-card-bg)] rounded-md items-center">
+  <div class="font-bold text-[var(--pd-invert-content-card-text)]">{title}</div>
+  <Button onclick={() => window.openExternal(link)}>Read more</Button>
+</div>
```

Comment:
issue: there is a missing EOF

Reply:
Fixed manually... then I just noticed it was reverted again.

I'll fix this again before next commit, but biome isn't flagging or fixing this.